drivers/sensor/pmw3360/pmw3360.c

```diff
@@ -290,48 +286,40 @@ static int reg_write(struct pmw3360_data *dev_data, uint8_t reg, uint8_t val)
 		.count = 1
 	};
 
-	err = spi_write(dev_data->spi_dev, &spi_cfg, &tx);
+	err = spi_write_dt(&dev_config->bus, &tx);
 	if (err) {
 		LOG_ERR("Reg write failed on SPI write");
 		return err;
 	}
 
 	k_busy_wait(T_SCLK_NCS_WR);
 
-	err = spi_cs_ctrl(dev_data, false);
```

Comment:
This should not be removed.

Reply:
Also in all other places where it was removed. We must control SPI's CS pin manually everywhere since we do not pass it to the SPI driver.